nilearn/glm/first_level/experimental_paradigm.py

```diff
@@ -17,6 +17,12 @@
 
 import numpy as np
 
+VALID_FIELDS = set(["name",
```

Comment:
is 'name' a valid field ? iIm quite surprised...

Reply:
All events creating functions in `test_paradigm.py` return a dataframe with a `name` column, for example:

https://github.com/nilearn/nilearn/blob/a8320c343487d49e0571e3dbcda6baab6d8cdb4c/nilearn/glm/tests/test_paradigm.py#L14-L21

Although `name` is not valid in the sense that it will be used, I didn't want to systematically raise warnings for this specific key. If you think this is a preferable behaviour, I can remove "name" from this set (and/or deal with it as a special case in the function `check_events` itself).

Reply:
This is an error that has been carried by history of the project. We must allow only BIDS compliant fields.